spec/resources/numismatics/issues/numismatics/issues_feature_spec.rb

```diff
@@ -196,6 +196,58 @@
     end
   end
 
+  context "with an existing reference" do
+    let(:reference1) { FactoryBot.create_for_repository(:numismatic_reference) }
+    let(:reference2) { FactoryBot.create_for_repository(:numismatic_reference, title: "Test Reference2") }
+
+    before do
+      reference1
+      reference2
+    end
+
+    scenario "when users add citations for an Issue", js: true do
+      visit edit_numismatics_issue_path(numismatic_issue)
+
+      # find("#numismatic_citation input.vs__search").send_keys("Test Reference", :enter)
```

Comment:
I could not manage to find any way to trigger this behavior - I spent some time attempting various combinations of CSS selectors, using Nokogiri to isolate the DOM elements, and using jQuery through https://rubydoc.info/github/jnicklas/capybara/master/Capybara/Session#evaluate_script-instance_method

I can remove these statements (or tests entirely), as I am not quite certain how to proceed with testing for this.

Reply:
Maybe remove the statements. Do you think the tests are useful without them? If not, I might remove them as well.  I have found doing JS testing of these components with Capybara nearly impossible.

Reply:
I can remove the statements, thank you for reviewing this.

There is one approach I might be able to propose for future PRs, using https://jestjs.io/docs/en/puppeteer, but I will delay this for future work.